Dockerfile

```diff
@@ -7,7 +7,9 @@ WORKDIR /app
 
 # Copy and build
 COPY ./src /app
-RUN dotnet publish /app/OrchardCore.Cms.Web -c Release -o ./build/release --framework net8.0
+COPY Directory.Build.props /
+
+RUN dotnet publish /app/OrchardCore.Cms.Web -c Release -o ./build/release --framework net8.0 /p:RunAnalyzers=false
```

Comment:
Why `/p:RunAnalyzers=false`? Only to not have `OrchardCore.ProjectTemplates` fail the build? Because I fixed that under https://github.com/OrchardCMS/OrchardCore/pull/15764/files#diff-72ab0c422f0f64ea126704874042eb9d59c62896d9045e23614c50edc58df923 (BTW please review), but otherwise, there should be no analyzer violations or any other build warnings.

Though it shouldn't matter much either way, since in this build we don't really need any further checks (that happens in the CI build already).

Reply:
The build fails since we treat warnings as errors. The docker image should not care about analyzers. As long as the code build we should be able to generate a usable image. Analyzers are for us to clean up out code and should pass before merging any core to main.

Reply:
Ah I see, because before https://github.com/OrchardCMS/OrchardCore/pull/15764 we had some `<TreatWarningsAsErrors>true</TreatWarningsAsErrors>` in csprojs.

Reply:
Yes. This may be removed in your PR. But if you remove it, make sure that the build still works "docker build ." Will build it assuming you have docker desktop running

Reply:
It's fine, since we still want to disable analyzers here.